packages/wp-now/src/get-sqlite-path.ts

```diff
@@ -6,5 +6,12 @@ import { SQLITE_FILENAME } from './constants';
  * The full path to the "SQLite database integration" folder.
  */
 export default function getSqlitePath() {
-	return path.join(getWpNowPath(), `${SQLITE_FILENAME}-main`);
+	return path.join(getWpNowPath(), 'mu-plugins', `${SQLITE_FILENAME}-main`);
```

Comment:
Cound `-main` be a part of `SQLITE_FILENAME`? Right now one has to remember to append it sometimes.

Reply:
Done